examples/tutorials/doc_examples/examples/array_editor.py

```diff
@@ -1,39 +0,0 @@
-#  Copyright (c) 2007, Enthought, Inc.
```

Comment:
Looks like these examples are getting deleted in this PR

Reply:
nope. this `array_editor` example got moved to `docs\source\traitsui_user_manual\examples\array_editor.py`. In some cases, git recognizes that I simply moved the file and made some changes. In other cases, git doesn't recognize this and thinks i deleted some files and created some from scratch - coincidentally with almost the same content.